provision/provision-site.sh

```diff
@@ -30,6 +30,26 @@ VVV_CONFIG=/vagrant/config.yml
 
 . "/srv/provision/provisioners.sh"
 
+# @description Retrieves a config value for the given site as specified in `config.yml`
+#
+# @arg $1 string the config value to fetch
+# @arg $2 string the default value
+function vvv_get_site_config_value() {
+  local value=$(shyaml -q get-value "sites.${SITE_ESCAPED}.${1}" "${2}" < ${VVV_CONFIG})
+  echo "${value}"
+}
+php_version=$(readlink -f /usr/bin/php)
```

Comment:
It's possible that a provisioner somewhere else either failed or interfered with this, so we should always change the PHP version used if it's not the default version, e.g. if a previous site switched the PHP version to 8.0 then failed it would test against 8.0 rather than the default of 7.4

Reply:
the next line does this, check if the php version is different from the default otherwise reset it